Components/button/button.html

```diff
@@ -0,0 +1,237 @@
+                                style="width: 1137px; height: 201px; border:0; transform: scale(1); overflow:hidden;"
+                                sandbox="allow-scripts allow-same-origin">
+                            </iframe>
+
+                        </div>
+
+
+                        <div class="button__floating">
+                            <h2>Floating Action Button</h2>
+                            <p>
+                                Example → class="btn btn-float"
+                            </p>
+                            <div class="button__floating-examples">
+                                <button class="btn btn-float float-up">
+                                    <span class="material-icons">arrow_upward</span>
+                                </button>
+                                <button class="btn btn-float">
+                                    <span class="material-icons">add</span>
+                                </button>
+                                <button class="btn btn-float">
+                                    <span class="material-icons">arrow_downward</span>
+                                </button>
+                            </div>
+                        </div>
+
+                        <div class="button__floating-iframe">
+                            <iframe
+                                src="https://carbon.now.sh/embed?bg=rgba%28255%2C255%2C255%2C0%29&t=paraiso-dark&wt=none&l=css&width=1137&ds=true&dsyoff=0px&dsblur=0px&wc=true&wa=false&pv=0px&ph=0px&ln=false&fl=1&fm=dm&fs=16.5px&lh=154%25&si=false&es=2x&wm=false&code=%253Cbutton%2520class%253D%2522btn%2520btn-float%2520float-up%2522%253E%250A%2520%2520%2520%2520%253Cspan%2520class%253D%2522material-icons%2522%253Earrow_upward%253C%252Fspan%253E%250A%253C%252Fbutton%253E%250A%253Cbutton%2520class%253D%2522btn%2520btn-float%2522%253E%250A%2520%2520%2520%2520%253Cspan%2520class%253D%2522material-icons%2522%253Eadd%253C%252Fspan%253E%250A%253C%252Fbutton%253E%250A%253Cbutton%2520class%253D%2522btn%2520btn-float%2522%253E%250A%2520%2520%2520%2520%253Cspan%2520class%253D%2522material-icons%2522%253Earrow_downward%253C%252Fspan%253E%250A%253C%252Fbutton%253E"
+                                style="width: 1137px; height: 303px; border:0; transform: scale(1); overflow:hidden;"
```

Comment:
carbon in line styling can be optimised width: 100% according to your container can make it responsive.
or you can add common class in your main css for this too.